packages/react-components/theme-designer/src/components/Sidebar/Form.tsx

```diff
@@ -46,15 +46,15 @@ const useStyles = makeStyles({
 
 export interface FormProps {
   sidebarId: string;
-  dispatchAppState: React.Dispatch<DispatchTheme>;
   formState: CustomAttributes;
   setFormState: React.Dispatch<CustomAttributes>;
 }
 
 export const Form: React.FC<FormProps> = props => {
   const styles = useStyles();
 
-  const { sidebarId, dispatchAppState, formState, setFormState } = props;
+  const { sidebarId, formState, setFormState } = props;
```

Comment:
_nit:_ Order these alphabetically

```suggestion
  const { formState, setFormState, sidebarId } = props;
```